walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
```

Comment:
Maybe keep only one getRandom button because they both have the same functionality just in different implementations.

Reply:
That makes sense - will do!

Reply:
Just realized this code is from the walkthrough.